SCons/Tool/javac.xml

```diff
@@ -278,27 +278,32 @@ env = Environment(JAVACCOMSTR="Compiling class files $TARGETS from $SOURCES")
     <cvar name="JAVAVERSION">
         <summary>
             <para>
-                Specifies the Java version being used by the &b-Java; builder.
-                This is <emphasis>not</emphasis> currently used to select one
-                version of the Java compiler vs. another.
-                Instead, you should set this to specify the version of Java
-                supported by your &javac; compiler.
-                The default is <literal>1.4</literal>.
-            </para>
-
-            <para>
+                Specifies the Java version being used by the &b-link-Java;
+                builder. Set this to specify the version of Java targeted
+                by the &javac; compiler.
                 This is sometimes necessary because
                 Java 1.5 changed the file names that are created
                 for nested anonymous inner classes,
                 which can cause a mismatch with the files
                 that &SCons; expects will be generated by the &javac; compiler.
-                Setting &cv-JAVAVERSION; to
-                <literal>1.5</literal>
-                (or <literal>1.6</literal>, as appropriate)
-                can make &SCons; realize that a Java 1.5 or 1.6
-                build is actually up to date.
+                Setting &cv-JAVAVERSION; to a version greater than
```

Comment:
Did you intend to change this from 1.5 -> 1.4?

Reply:
I changed it from "1.5 or 1.6" to "a version greater than 1.4" to avoid enumerating all the versions that now exist beyond 1.4. That's ok, no?  Really this "default to 1.4" needs to change - 1.4 was a 2002 release, surely we'll never run into that.  Should I add an "impending change" notice to the RELEASE file? Or somewhere?

Reply:
Good point.  
How old a release is a reasonable default? 1.8?  I still see that in use in the wild.
I think that's fine and just put a blurb in RELEASE.txt CHANGED/...
I don't think we need a deprecation to change from an obsolete default.

Reply:
1.7/1.8 is pretty reasonable.  1.8 is still receiving updates, having become "the last LTS before everything changed".